core/vmpu/src/armv7m/vmpu_armv7m.c

```diff
@@ -300,79 +300,97 @@ void vmpu_load_box(uint8_t box_id)
 
 extern int vmpu_region_bits(uint32_t size);
 
+/* Compute the MPU region size for the given BSS sections and stack sizes.
+ * The function also updates the region_start parameter to meet the alignment
+ * requirements of the MPU. */
+static uint32_t vmpu_acl_stack_region_size(uint32_t * const region_start, uint32_t const bss_size,
```

Comment:
I would have preferred to keep `uint32_t * const region_start` (*region_start is a constant pointer to uint32_t*), but I see your point on readability.

Fixed.